core/src/client/async_client/mod.rs

```diff
@@ -20,8 +20,9 @@ use helpers::{
 	build_unsubscribe_message, call_with_timeout, process_batch_response, process_notification,
 	process_single_response, process_subscription_response, stop_subscription,
 };
-use jsonrpsee_types::{ResponseSuccess, TwoPointZero};
+use jsonrpsee_types::{InvalidRequestId, ResponseSuccess, TwoPointZero};
 use manager::RequestManager;
+use std::sync::Arc;
 
 use async_lock::Mutex;
```

Comment:
I wonder whether its worth doing `use async_lock::Mutex as AsyncMutex` to make it clear in other places that this is not a sync Mutex?